modules/luci-base/po/zh-tw/base.po

```diff
@@ -1,18 +1,21 @@
+#
+# Yangfl <mmyangfl@gmail.com>, 2018.
+#
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: 2014-05-21 10:34+0200\n"
-"Last-Translator: omnistack <omnistack@gmail.com>\n"
-"Language-Team: none\n"
+"PO-Revision-Date: 2018-08-02 13:34+0800\n"
+"Last-Translator: Yangfl <mmyangfl@gmail.com>\n"
+"Language-Team:  <debian-l10n-chinese@lists.debian.org>\n"
 "Language: zh_TW\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
-"X-Generator: Pootle 2.0.6\n"
+"X-Generator: Gtranslator 2.91.7\n"
 
 msgid "%.1f dB"
-msgstr ""
+msgstr "%.1f dB"
```

Comment:
This change can be safely left out.  Original text will be retained if translation does not exist.  Very likely we need to modify source code to unmark this as translation-needed

Reply:
just filling untranslated strings, but nah.